base/generate-ci-config

```diff
@@ -16,14 +16,17 @@ export DEBIAN_FRONTEND=noninteractive
 apt-get update -y -qq > /dev/null
 
 # Generate a list of all kernel versions which have NVIDIA precompiled driver packages available.
-SUPPORTED_KERNELS=$(apt-cache search linux-objects-nvidia-${DRIVER_BRANCH}-server.*${LTS_KERNEL}.*${KERNEL_FLAVOR} | awk '{print $1}' | grep -v "open" | sed -e "s/^.*${LTS_KERNEL}/${LTS_KERNEL}/" | sort -n -t'-' -k2)
+SUPPORTED_KERNELS_LINUX_SIGNATURES_LIST=$(apt-cache search "linux-signatures-nvidia.*${LTS_KERNEL}.*${KERNEL_FLAVOR}" | awk '{print $1}'  | sed -e "s/^.*${LTS_KERNEL}/${LTS_KERNEL}/" | sort -n -t'-' -k2| grep "${KERNEL_FLAVOR}$")
```

Comment:
Question: what is the rationale for using the `linux-signatures-nvidia` package instead of the `linux-objects-nvidia` package?

Reply:
I had suggested this earlier as the `linux-objects-nvidia` package is only specific to the proprietary modules whereas `linux-signatures-nvidia` is used in both. 

I didn't realise that we were looking up `linux-objects-nvidia`, so one could argue that this particular change can be made in a follow-up PR

Reply:
Ah okay. I think I am okay with using the `linux-signatures-nvidia` package. Was just curious what led to this change.